API/models/model.go

```diff
@@ -124,6 +124,29 @@ func PropagateParentIdChange(ctx context.Context, oldParentId, newId string, ent
 	return nil
 }
 
+// PropagateDomainChange: search for all objects with reference to the modified domain
+func PropagateDomainChange(ctx context.Context, oldDomainId, newDomainId string) error {
+	// Find all objects containing this domain
+	req := bson.M{"domain": primitive.Regex{Pattern: "^" + oldDomainId + "(\\" + u.HN_DELIMETER + "|$)", Options: ""}}
+	// For each object found, replace old domain by new
+	update := bson.D{{
```

Comment:
Personally, I prefer that the closing braces are also indented, example:
update := bson.D{{
		Key: "$set",
		Value: bson.M{
			"domain": bson.M{
				"$replaceOne": bson.M{
					"input": "$domain",
					"find": oldDomainId,
					"replacement": newDomainId,
				},
			},
		},
	}}

Reply:
I prefer more compact there 😅 thought it was easier to read